nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/ProcessGroupDTO.java

```diff
@@ -200,4 +204,13 @@ public void setInactiveRemotePortCount(Integer inactiveRemotePortCount) {
         this.inactiveRemotePortCount = inactiveRemotePortCount;
     }
 
+
+    @ApiModelProperty("The variables that are configured for the Process Group")
+    public Map<String, String> getVariables() {
+        return variables;
+    }
+
+    public void setVariables(final Map<String, String> variables) {
```

Comment:
Also, can we update the documentation to indicate that this only includes the variables defined at this Process Group?